Src/AutoFakeItEasy/FakeItEasyRelay.cs

```diff
@@ -82,13 +82,28 @@ public object Create(object request, ISpecimenContext context)
 
             var fakeType = typeof(Fake<>).MakeGenericType(type);
 
-            var fake = context.Resolve(fakeType) as FakeItEasy.Configuration.IHideObjectMembers;
+            var fake = context.Resolve(fakeType);
             if (fake == null)

```

Comment:
Being unsure of how the pieces fit together, I was being overly cautious. I'll remove the extra work, in a way that mirrors the change you asked for in `FakeItEasyBuilder`.